<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsForm</class>
 <widget class="QWidget" name="StatisticsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Year from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="yearFrom">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Year to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="yearTo">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Section:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sectionList"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>764</width>
        <height>436</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGraphicsView" name="statisticsArea">
         <property name="renderHints">
          <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnCsv">
       <property name="text">
        <string>CSV Export</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/mime-csv.png</normaloff>:/icons/resources/icons/mime-csv.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnExport">
       <property name="text">
        <string>PDF Export</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/mime-pdf.png</normaloff>:/icons/resources/icons/mime-pdf.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPrint">
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/printer.png</normaloff>:/icons/resources/icons/printer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../PiTres.qrc"/>
 </resources>
 <connections/>
</ui>
